src/main/java/net/glowstone/block/ItemTable.java

```diff
@@ -301,6 +299,31 @@ private void registerBuiltins() {
         reg(Material.HOPPER_MINECART, new ItemMinecart(GlowMinecart.MinecartType.HOPPER));
         reg(Material.POWERED_MINECART, new ItemMinecart(GlowMinecart.MinecartType.FURNACE));
         reg(Material.STORAGE_MINECART, new ItemMinecart(GlowMinecart.MinecartType.CHEST));
+        
+        //Wearable Items
+        reg(Material.LEATHER_BOOTS, new ItemType().wear(1, ItemWearablePosition.FEET, true));
```

Comment:
Well items such as heads, pumpkins, and elytras can be work but cannot be enchanted. The integer is to determine how many armor points wearing that item gives you. 

>  Would also need a "value" level to determine which armor material is better than another.

Then for where you wear this item and then if you should be able to equip this item by right clicking it.